app/views/handlers/list_remove_question.py

```diff
@@ -21,6 +21,14 @@ def is_location_valid(self):
         return True
 
     def handle_post(self):
+        answer_block = self._schema.get_add_block_for_list_collector(  # pylint: disable=protected-access
+            self.parent_block["id"]
+        )
+        answer_id = self._schema.get_first_answer_id_for_block(answer_block["id"])
```

Comment:
Why the first answer?.

Reply:
There is no method in schema class to extract the only answer id in a question block so I found this one useful.

Reply:
Moved it to `remove_list_item_and_answers` but still using this method.

Reply:
This one is removed now.